product_docs/docs/bdr/4/conflicts.mdx

```diff
@@ -186,13 +183,13 @@ Updating the Primary Key column is possible, so this SQL succeeds:
 UPDATE pktest SET pk=2 WHERE pk=1;
 ```
 
-...but if we have multiple rows in the table, e.g.:
+However, suppose there are multiple rows in the table:
 
 ```sql
 INSERT INTO pktest VALUES (3,3);
 ```
 
-...then some UPDATEs would succeed:
+Some UPDATE operations succeed:
```

Comment:
If we start replacing "UPDATEs" and "INSERTs" and "DELETEs" with "UPDATE operations", we'll need to be more context aware, I am afraid. For example, I "operation" makes sense when we are talking about the replicat (the apply side of the replication), but when we talk about SQL (user input) it's not "UPDATE operation" but "UPDATE statement".

So i.e. the 'INSERT operations that violate multiple UNIQUE constraints" above makes sense, "Where two concurrent `UPDATE` operations on different nodes change the same tuple" also above, I am not quite sure whether it should be "operation" or "statement" (which means likely both are fine) and the "Some UPDATE operations succeed" here should definitely be "statement".

Reply:
Then the question is whether we are making it more readable or more confusing, of course :)

Reply:
I'll revert the changes. Better to avoid the possibility of confusion if the rules about when to use statement vs operation are too hard to figure out.